core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -101,23 +105,28 @@ function (string $value): string {
                 ->end()
                 ->booleanNode('pretty_error_screens')
                     ->defaultValue(false)
+                    ->info('Enables pretty error screens, for which custom templates can be created.')
                 ->end()
                 ->integerNode('error_level')
+                    ->info('The error reporting level set when the framework is initialized. Defaults to E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_USER_DEPRECATED.')
                     ->min(-1)
                     ->max(32767)
                     ->defaultValue(E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_USER_DEPRECATED)
                 ->end()
                 ->arrayNode('locales')
                     ->prototype('scalar')->end()
+                    ->info('Allows to configure which languages can be used within Contao. Defaults to all languages for which a translation exists.')
```

Comment:
Isn't this for the back end only?

Reply:
Not sure, I haven't actually checked yet.

Reply:
The `LocaleListener` uses it too, so it is technically used for the front end as well.

Reply:
It is used system-wide.